I-D-Profile-Negotiation.xml

```diff
@@ -156,13 +159,16 @@
 					Examples of a profile in this context include, but are not limited to,
 					Dublin Core application profiles
 					(formally expressed in <xref target="DSP">Description Set Profiles (DSP)</xref>),
-					<xref target="XSD">XML Schema documents</xref>,
-					and RDF Shapes expressed in <xref target="SHACL">SHACL</xref>.
+					<xref target="W3C.REC-xmlschema11-1-20120405">XML Schema documents</xref>,
+					and RDF Shapes expressed in <xref target="W3C.REC-shacl-20170720">SHACL</xref>.
 					Profiles can, but do not need to, be independent of a media type.
 					The definition and usage of profiles in applications
 					is beyond the scope of this proposal;
-					our only constraint is that a profile can be identified by a URI.
-					Typical use cases include validation,
+					our only constraint is that profiles are identified by URIs.
```

Comment:
I think the original language was clearer here. (singular/plural)

Reply:
I'll happily change back. My main point was to change "can be identified" to "is identified" since the header syntax requires the use of URIs.